src/common/imageio_module.h

```diff
@@ -231,6 +231,12 @@ int dt_imageio_get_index_of_storage(dt_imageio_module_storage_t *storage);
 /* add a module into the known module list */
 void dt_imageio_insert_storage(dt_imageio_module_storage_t *storage);
 
+// This function generates the a and b fraction's elements to calculate the scaling factor
```

Comment:
you probably also to fix a & b in comment.

Reply:
Yes, of course I will fix it. I'm sorry.